client/errors.go

```diff
@@ -61,6 +63,19 @@ func classifyError(err error, fallbackType diag.Type, accounts []Account, opts .
 				),
 			}
 		}
+		if ae.ErrorMessage() == ssoInvalidOrExpired {
+			return diag.Diagnostics{
+				RedactError(accounts, diag.NewBaseError(err,
+					diag.ACCESS,
+					append(opts,
+						diag.WithType(diag.ACCESS),
+						diag.WithSeverity(diag.WARNING),
+						ParseSummaryMessage(err),
+						diag.WithDetails(errorCodeDescriptions[ae.ErrorCode()]),
```

Comment:
add `AuthFailure` to `errorCodeDescriptions`?

Reply:
added